pkg/northbound/proto/diags.proto

```diff
@@ -30,6 +30,11 @@ message ConfigRequest {
     repeated string deviceIds = 1;
 }
 
+// OpStateRequest is a message for specifying GetOpState query parameters.
+message OpStateRequest {
+    repeated string deviceId = 1;
```

Comment:
I've changed my mind on this - can you remove the **repeated** term - a single one is better. It shows that at least one is required - there can be so much for one device, we would not want more than one at a time